get_alerts.py

```diff
@@ -6,10 +6,55 @@
 import csv
 import os
 import requests
+import re
 
 API_ENDPOINT = 'https://app.threatstack.com/api/v1/alerts'
 
 TIMESTAMP_FIELDS = ['created_at', 'expires_at', 'last_updated_at']
+
+def main():
+    '''
+    Main method. Collect command line arguments, and pass them into the get_alerts method
+    '''
+    PARSER = argparse.ArgumentParser(description='Query threatstack alerts.')
```

Comment:
Does this code style still pass pylint? Pylint may want you to lowercase parser now that it is in a dedicated method as opposed to in the global namespace.